### ISSUE TITLE
Trying to mock in Meteor without success.

### ISSUE THREAD

**Issue by dcusan** [theKashey/rewiremock#9]
Hi again @theKashey,

I am trying to use rewiremock with the [Meteor](https://github.com/meteor/meteor) JS framework without much success.

I imagine you wont be able to offer any help in this regard but I though I'd try :)

 I've created a small reproduction of my issue in this [repo](https://github.com/dcusan/meteor-rewiremock).

Meteor uses Babel under the hood to transpile `import` statements into `require` statements however there must be something else going on under the hood as my test is still failling.

You should be able to see this when running the `npm test` command after installing Meteor. As you can see we have our own test runner in order to load Meteor globals among others but it is still mocha.

No matter the outcome, thanks for your help.

**Comment by theKashey** [comment]
Same :(
What I found:
1. rewiremock's internals are not called at all. It does not do anything. This means that `rewiring` mechanics (hook on Module._load) are not the case for Meteor, as it not the case for Webpack.
2. Meteor are doing some magic the `real` file name is
`module.dynamicImport('./moduleB.js'):/private/var/folders/r2/cr_bp76j4ps6twpj73zys3s509l19f/T/meteor-test-run1x7v55f.qe09g/.meteor/local/build/programs/server/npm-require.js`
3. The current version of WebStorm could not debug meteor apps (version==1.6). So after the next release I will able to track problem down, and, maybe, fix it. But not now.

PS: Even if rewiremock will work - you test case will not work. There is no way to "hot-rewire" imported module.
First setup rewires, next enable rewiremock, last - require the target module. Or use rewiremock.proxy/around to do the same. You have to import target module AFTER setting up rewires.

```js
it('lets moduleB be mocked', async () => {
    //addPlugin(plugins.relative)
    rewiremock( () => require('./moduleB.js')) // use "require" to remove need of plugins
      .with({
        doB: () => 'A'
      })
      .toBeUsed(); // <-- just always use this command :)

    const modA = require('./moduleA.js')

    expect(modA.doA()).to.equal('AA')

   // removePlugin(plugins.relative)
  })
```

**Comment by dcusan** [comment]
Thank you for looking into this and the tip regarding the tests! 🙏

I understood now that only after having setup `rewiremock` I can `require` the module under test and expect it to be mocked.
